packages/core/src/icon/icon.stories.ts

```diff
@@ -88,8 +88,8 @@ export const all = () => {
     'lg',
     propertiesGroup
   );
-  const dir = select(
-    'dir',
+  const direction = select(
```

Comment:
Why is this change needed?

Reply:
Sorry, undesired commits came along, fixed it.